src/layouts/AppContainer.tsx

```diff
@@ -163,31 +184,9 @@ class AppContainer extends React.Component<IProps, IState> {
 
           <div className={css.pendingTransactions}>
             {this.state.notificationsMinimized ?
-              <MinimizedNotifications
-                notifications={sortedNotifications.length}
-                unminimize={() => this.setState({notificationsMinimized: false})}
-              /> :
-              sortedNotifications.map(({id, status, title, message, fullErrorMessage, timestamp, url}): any => (
-                <div key={id}>
-                  <Notification
-                    title={(title || status).toUpperCase()}
-                    status={
-                      status === NotificationStatus.Failure ?
-                        NotificationViewStatus.Failure :
-                        status === NotificationStatus.Success ?
-                          NotificationViewStatus.Success :
-                          NotificationViewStatus.Pending
-                    }
-                    message={message}
-                    fullErrorMessage={fullErrorMessage}
-                    url={url}
-                    timestamp={timestamp}
-                    dismiss={() => dismissNotification(id)}
-                    showNotification={showNotification}
-                    minimize={() => this.setState({notificationsMinimized: true})}
-                  />
-                </div>
-              ))
+              <MinimizedNotifications notifications={sortedNotifications.length} unminimize={this.unminimizeNotif()} />
+              :
+              sortedNotifications.map(this.sortedNotification())
```

Comment:
this looks weird, maybe name the function something different?

Reply:
done